examples/anime-http/server/auth/trendingAnimeServer.ts

```diff
@@ -0,0 +1,28 @@
+// /* eslint-disable no-undef */
+
+// //@ts-ignore
+// import { Message } from "@asyncapi/glee";
+import axios from "axios"
+
+export async function serverAuth({ headers, callback: done }) {
+  await axios.get("https://jsonplaceholder.typicode.com/todos/1", {
+    timeout: 5000,
+  })
+
```

Comment:
```suggestion
export async function serverAuth({ params, callback: done }) {
  await axios.get("https://jsonplaceholder.typicode.com/todos/1", {
    timeout: 5000,
  })
  
  params.getParams()
  // or
  params() it could be a function that just return the parameters that are needed. 

```

and we can parse the security requirement to just return what is needed.